pkg/reconciler/reconciler.go

```diff
@@ -70,8 +70,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, pr *v1beta1.PipelineRun)
 
 		provider, event, err := r.detectProvider(ctx, logger, pr)
 		if err != nil {
-			logger.Error(err)
-			return nil
+			return fmt.Errorf("detectProvider: %w", err)
```

Comment:
Thanks for this changes for errors :)
this was in my mind to do and did in my local repo .. 😄